glide.yaml

```diff
@@ -60,6 +60,9 @@ import:
   - package: github.com/gogo/protobuf
     version: ^1
 
+  - package: github.com/jhump/protoreflect
+    version: ^1.2
```

Comment:
Can you pin this commit please: 6feecbc20c4c369d6494c59ac5aee5d0df3c9d80

Included an important bug fix and my new method: https://github.com/jhump/protoreflect/commit/6feecbc20c4c369d6494c59ac5aee5d0df3c9d80

Reply:
sure